### PR TITLE
ignore carto python warnings

### PR DESCRIPTION
**Pull request by oleurud**
Solves a part of https://github.com/CartoDB/cartoframes/issues/1090

**CR and merge https://github.com/CartoDB/carto-python/pull/154 first**.  ~Then, I will remove the `filterwarnings` from `api_context` file~ (done)

### CODE REVIEW

File: cartoframes/lib/context/api_context.py [hunk 1]
```diff
@@ -10,6 +10,9 @@
 from .base_context import BaseContext
 from ...__version__ import __version__
 
+from warnings import filterwarnings
```
Comment (oleurud) [inline cartoframes/lib/context/api_context.py]: Since we are going to remove the comment in `carto-python`, after merging it, we could avoid this changes

Comment (oleurud) [reply cartoframes/lib/context/api_context.py]: It is already removed in carto-python. So, I have removed these 2 lines because they are not needed

**Comment by oleurud** [comment]
Auto acceptance :green_apple: 

image

File: cartoframes/data/dataset/dataset_info.py [hunk 2]
```diff
@@ -9,6 +9,9 @@
 from ...utils.geom_utils import setting_value_exception
 from ...utils.columns import normalize_name
 
+from warnings import filterwarnings
```
Comment (alasarr) [inline cartoframes/data/dataset/dataset_info.py]: Is that still necessary after removing warnings in CARTO Python?

Comment (oleurud) [reply cartoframes/data/dataset/dataset_info.py]: Yes, it is. In this case, to get the info about datasets from a CARTO account (we need to get the privacy), we are using a non public API. In the carto-python code we are raising a `FutureWarning` warning about that. This is what we are filtering here.

Comment (oleurud) [reply cartoframes/data/dataset/dataset_info.py]: By the way, we have some of the spread by the CARTOframes code

**Pull request merged**